### PR TITLE
Add documentation on lsp6-signer package

### PR DESCRIPTION
**Pull request by magalimorin18**

### CODE REVIEW

File: docs/tools/lsp6-signerjs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 1.1
```
Comment (Hugoo) [inline docs/tools/lsp6-signerjs/getting-started.md:2]: no need to have float here - only integers.

There was this thing that some people started using float numbers and then every one started putting floats everywhere. It is not needed.

```suggestion
sidebar_position: 1
```

File: docs/tools/lsp6-signerjs/Classes/LSP6Signer.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  {
+    message: 'hello',
+    messageHash:
+      '0xd8fd712232755d5f8819c95fdd911dcd51c012b8a66f9710ef0c86ad30925fa6',
+    v: '0x1c',
+    r: '0x1eab2de0103b8e82650f9706b17cf2adce55a335e7041bad5a94ab49c56a9c12',
+    s: '0x662e80a369ffa2a6a77fbeaad1f32653cbd74860c8fbc999b1fc47b8d1cb7d93',
+    signature:
+      '0x1eab2de0103b8e82650f9706b17cf2adce55a335e7041bad5a94ab49c56a9c12662e80a369ffa2a6a77fbeaad1f32653cbd74860c8fbc999b1fc47b8d1cb7d931c',
+  };
```
Comment (Hugoo) [inline docs/tools/lsp6-signerjs/Classes/LSP6Signer.md:73]:
```suggestion
lsp6Signer.sign(
  'hello',
  'ffeb17b9a6059fec3bbab63d76b060b7380cac7a62ce6621a134531a46458968',
);

/**
  {
    message: 'hello',
    messageHash:
      '0xd8fd712232755d5f8819c95fdd911dcd51c012b8a66f9710ef0c86ad30925fa6',
    v: '0x1c',
    r: '0x1eab2de0103b8e82650f9706b17cf2adce55a335e7041bad5a94ab49c56a9c12',
    s: '0x662e80a369ffa2a6a77fbeaad1f32653cbd74860c8fbc999b1fc47b8d1cb7d93',
    signature:
      '0x1eab2de0103b8e82650f9706b17cf2adce55a335e7041bad5a94ab49c56a9c12662e80a369ffa2a6a77fbeaad1f32653cbd74860c8fbc999b1fc47b8d1cb7d931c',
  }
 
*/
```

**Review by Hugoo — CHANGES_REQUESTED** [review]

File: CODEOWNERS [hunk 3]
```diff
@@ -1,17 +1,23 @@
...
 # Each line is a file pattern followed by one or more owners.
 
 # These owners will be the default owners for everything in
+
 # the repo. Unless a later match takes precedence, @Hugoo will be requested for
+
 # review when someone opens a pull request.
-*   @Hugoo
+
+- @Hugoo
```
[the remark this answers is not in the document]
Comment (magalimorin18) [reply CODEOWNERS:11]: Seems prettier made it as I didnt touch this line

File: docs/tools/lsp6-signerjs/Classes/LSP6Signer.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 1.1
+---
+
+# LSP6Signer
+
+# Usage
```
[the remark this answers is not in the document]
Comment (magalimorin18) [reply docs/tools/lsp6-signerjs/Classes/LSP6Signer.md:7]: What did you want to change ?

File: CODEOWNERS (hunk 3 above)
[the remark this answers is not in the document]
Comment (magalimorin18) [reply CODEOWNERS:11]: Yes verifie, when I save the file it changes * to -

**Review by Hugoo — APPROVED** [review]

**Pull request merged**